### ISSUE TITLE
Implement <cfloat>

### ISSUE THREAD

**Issue by mysterymath** [llvm-mos/llvm-mos-sdk#58]

**Comment by atirut-w** [comment]
Since this is a library for float numbers, shouldn't support for `float`s be added first?

**Comment by mysterymath** [comment]
Yep, but there isn't a github feature to track deps between bugs, unfortunately. Although I've recently started using the issue mention feature to track this implicitly.
